SpatialGDK/Source/Public/SpatialGDKLoader.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Paths.h"
+#include "PlatformProcess.h"
+
+#include "IPluginManager.h"
+
+/**
+ * This class ensures that the C API worker library is loaded before it is needed by code.
+ */
+class FSpatialGDKLoader
+{
+public:
+	FSpatialGDKLoader()
+	{
+#if PLATFORM_WINDOWS
+		FString Path = IPluginManager::Get().FindPlugin(TEXT("SpatialGDK"))->GetBaseDir() / TEXT("Binaries/ThirdParty/Improbable");
+
+#if PLATFORM_64BITS
+		Path = Path / TEXT("Win64");
+#else
+		Path = Path / TEXT("Win32");
+#endif
+		Path = Path / TEXT("worker.dll");
+		WorkerLibraryHandle = FPlatformProcess::GetDllHandle(*Path);
```

Comment:
Should we add a path check here, and if it fails tell the user to run Setup.bat?